catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/forms/model/JsonModel.java

```diff
@@ -0,0 +1,272 @@
+package org.codice.ddf.catalog.ui.forms.model;
+
+import static org.codice.ddf.catalog.ui.util.SafelyAccess.safeGet;
+
+import ddf.catalog.data.Metacard;
+import ddf.catalog.data.impl.types.CoreAttributes;
+import java.io.Serializable;
+import java.util.Date;
+import java.util.List;
+import java.util.Set;
+import javax.xml.bind.JAXBElement;
+import net.opengis.filter.v_2_0.BinaryComparisonOpType;
+import net.opengis.filter.v_2_0.BinaryLogicOpType;
+import net.opengis.filter.v_2_0.LiteralType;
+import net.opengis.filter.v_2_0.LogicOpsType;
+import net.opengis.filter.v_2_0.ObjectFactory;
+import org.boon.json.annotations.JsonIgnore;
+import org.boon.json.annotations.JsonProperty;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class JsonModel {
+  private static final Logger LOGGER = LoggerFactory.getLogger(JsonModel.class);
+
+  private JsonModel() {
+    // Should not be instantiated
+  }
+
+  public static class Described {
+    private final String id;
+
+    private final String title;
+
+    private final String description;
+
+    private final Date created;
+
+    private final String owner;
+
+    public Described(Metacard metacard) {
+      this.id = safeGet(metacard, CoreAttributes.ID, String.class);
+      this.title = safeGet(metacard, CoreAttributes.TITLE, String.class);
+      this.description = safeGet(metacard, CoreAttributes.DESCRIPTION, String.class);
+      this.created = safeGet(metacard, CoreAttributes.CREATED, Date.class);
```

Comment:
I think we want this to be `Core.METACARD_CREATED` and for these references we can just use the interface directly `Core.*`